### ISSUE TITLE
Dropout layer in the network?

### ISSUE THREAD

**Issue by kuanfang** [ethereon/caffe-tensorflow#30]
I wonder what is the best way to use the dropout layer in the Network class if I want to load a pretrained Caffe model and fine tune on another task with dropout. I found that the converted network does not contains dropout layers. 

Currently I just define a keep_prob variable and manually insert dropout layers into the converted network, which is kind of messy. What is the expected way of using dropout layer in your original design?

**Comment by ethereon** [comment]
Actually, originally there was no dropout layer in `Network`. It got added as [part of a pull request] (https://github.com/ethereon/caffe-tensorflow/blame/master/kaffe/tensorflow/network.py) 

A corresponding mapper exists in `TensorFlowMapper`, should everything should work. Did you try setting the phase to train? (`./convert.py .... --phase=train`)

**Comment by kuanfang** [comment]
Thanks.
